docs/README.md

```diff
@@ -30,28 +30,79 @@ If you're interested in contributing to the SOK open source project, review the
 
 Review the [Change Log](ChangeLog.md) page for a history of changes in each release.
 
-
 ## Prerequisites for the Splunk Operator
 
-While we are only able to test and support a small subset of configurations, the Splunk Operator should work with any CNCF certified distribution of Kubernetes, version 1.12 or later. Setting up, configuring and managing Kubernetes clusters is outside the scope of this guide and Splunk’s coverage of support. You can submit bugs to https://github.com/splunk/splunk-operator/issues.
+### Supported Kubernetes Versions
+
+- Kubernetes 1.12 and later
+
+The Splunk Operator is supported with [CNCF certified](https://www.cncf.io/certification/software-conformance/) distributions of Kubernetes, version 1.12* and later.
+
+Implementing, configuring, and administering Kubernetes clusters is outside the scope of this guide, and Splunk’s coverage of support.
+
+You can submit defects to https://github.com/splunk/splunk-operator/issues.
+
+### Kubernetes Platform Suggestions:
+
+In order to create Splunk deployments through Splunk operator, access to a functioning kubernetes environment is required. Some options include:
+
+
+
+| Platform        | Splunk Smartstore Support | Additional Info|
+| ---------- | ------- | ------- |
+| [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks)| Yes | Actively used in the development of the Splunk Operator.|
+| [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine)| No | Actively used in the development of the Splunk Operator.|
+| [Kind](https://kind.sigs.k8s.io/) |  Yes| Actively used in the development of the Splunk Operator. Suitable for development/test environments only|
+| [Microsoft Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/)| No| No additional info available |
+| [Red Hat OpenShift](https://www.openshift.com/)| No| No additional info available|
+| [HPE Ezmeral](https://www.hpe.com/us/en/solutions/container-platform.html)|No | No additional info available|
+
 
 *Kubernetes releases 1.16.0 and 1.16.1 contain a
 [critical bug(https://github.com/kubernetes/kubernetes/pull/83789) that can
 crash your API server when using custom resource definitions. Do not
 attempt to run the Splunk Operator using these releases. This bug is fixed in
 Kuberenetes 1.16.2.*
```

Comment:
nit: not technically related to this PR, but typo "Kuberenetes" -> "Kubernetes" 
perhaps we can address it here

Reply:
The Kuberenetes still exists in the latest changes, however the rest of the changes look good to me and this is not related to your change.